### ISSUE TITLE
8 times downscale

### ISSUE THREAD

**Issue by volcanoxs** [pkuxmq/Invertible-Image-Rescaling#10]
Hi, thanks for your work.
The results of IRN4X and IRN2X are amizing.I really want to know if you have ever trained a  IRN8X network?Did it work well too?Do you have any suggestions to train a  IRN8X network.

**Comment by pkuxmq** [comment]
Yes, IRN_8x can work well. I have tried to just use three Downscaling Modules, keeping other details the same as 2x and 4x, and the results are also good.

**Comment by volcanoxs** [comment]
will the IRN8x model release later？Thanks

**Comment by pkuxmq** [comment]
It will be released later.
